pom.xml

```diff
@@ -572,5 +572,101 @@
+                <additionalOption>/java/_book.yaml</additionalOption>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>project.path</additionalOption>
+                <additionalOption>/_project.yaml</additionalOption>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>devsite.path</additionalOption>
+                <additionalOption>/java/docs/reference/</additionalOption>
+                <additionalOption>-d</additionalOption>
+                <additionalOption>${project.build.directory}/devsite</additionalOption>
+                <additionalOption>-templatedir</additionalOption>
+                <additionalOption>${devsite.template}</additionalOption>
+                <additionalOption>-toroot</additionalOption>
+                <additionalOption>/java/docs/reference/</additionalOption>
+                <additionalOption>-yaml</additionalOption>
+                <additionalOption>_toc.yaml</additionalOption>
+                <additionalOption>-warning</additionalOption>
+                <additionalOption>101</additionalOption>
+              </additionalOptions>
+              <useStandardDocletOptions>false</useStandardDocletOptions>
+            </configuration>
+          </plugin>
+          <plugin>
+            <!-- Clean up some references and files. -->
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.7</version>
+            <executions>
+              <execution>
+                <phase>site</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target>
+                    <echo message="Updating relative links in API docs" />
+                    <!-- TODO: What is the right behavior for io* and google*? -->
+                    <replace dir="${project.build.directory}/devsite" token="href=&quot;com" value="href=&quot;/java/docs/reference/com" />
+                    <replace dir="${project.build.directory}/devsite" token="href=&quot;io" value="href=&quot;/java/docs/reference/io" />
+                    <replace dir="${project.build.directory}/devsite" token="href=&quot;google" value="href=&quot;/java/docs/reference/google" />
+                    <copy file="${project.build.directory}/devsite/assets/_toc.yaml" todir="${project.build.directory}/devsite/reference" />
+                    <echo message="Removing files not needed by Devsite" />
+                    <delete file="${project.build.directory}/devsite/reference/classes.html" />
+                    <delete file="${project.build.directory}/devsite/reference/hierarchy.html" />
+                    <delete file="${project.build.directory}/devsite/reference/index.html" />
+                    <delete file="${project.build.directory}/devsite/reference/lists.js" />
+                    <delete file="${project.build.directory}/devsite/reference/package-list" />
```

Comment:
We will want to keep `package-list` available for cross-linking between javadocs. This is the file that is referenced when the javadoc command is provided `-link` option [1].

[1] https://docs.oracle.com/javase/9/javadoc/javadoc-command.htm#JSJAV-GUID-3894CFD5-88D5-407F-B603-3DCC826AE6EA

Reply:
Good catch, and good to know! I removed this line.